Detective notebook

JenTzori opened the pull request:
I reorganized the page to make room for the detective notebook. User may not interact with the detective notebook; it is updated automatically:
1. on page load, cards in user's hand are checked off
2. on refuted suggestion, card refuted is checked off

When you hover over a checkbox, you see the message "Source: my hand", "Source: other player", or "No evidence found yet". I didn't see the user ID of the refuter, do we have that? If so I can update the "Source: other player" message to identify the player. 

If y'all want, I can make it a popover (https://www.w3schools.com/bootstrap/bootstrap_popover.asp) rather than a tooltip so that it's more obvious.

I messed up Kelly's method of putting player tokens in rooms when I reorganized the page, so I'll fix that next (future PR).

Screenshot of new layout:
![image](https://user-images.githubusercontent.com/22458598/57151959-07c6ea80-6d87-11e9-991a-ae658c13cdf7.png)

JenTzori commented:
Updated screenshot:
![image](https://user-images.githubusercontent.com/22458598/57165838-188a5700-6dad-11e9-9039-467318ba3229.png)

Still need some more work on player token positioning, but this is as far as I'll get today

megmello commented:
Looks fantastic.  Thanks for sharing your progress.  I am working on my
pieces tonight.

On Fri, May 3, 2019 at 5:10 PM JenWindsor <notifications@github.com> wrote:

> Updated screenshot:
> [image: image]
> <https://user-images.githubusercontent.com/22458598/57165838-188a5700-6dad-11e9-9039-467318ba3229.png>
>
> Still need some more work on player token positioning, but this is as far
> as I'll get today
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/dfpreston/Clue-less-Colonel-Mustard-group-/pull/2#issuecomment-489240894>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALPGI3GEB2KCZRZYOFWWVDDPTSS3RANCNFSM4HKVMAJQ>
> .
>

JenTzori committed:
Added suspect descriptions


Using bootstrap design elements for gameplay options


Implemented detective notebook


Added pictures for rooms




Added back player pictures
